runtimes/pythonrt/runner/tee.py

```diff
@@ -0,0 +1,51 @@
+from queue import Queue
+from threading import Thread
+from time import sleep
+import grpc
+import pb
+
+
+class Tee:
+    def __init__(self, stream, runner_id, worker):
+        self.stream = stream
+        self.queue = Queue()
```

Comment:
I'm not an expert with this code, so this is more of a question that a comment:

Any queue is by definition a bottleneck, but (hopefully) a necessary one.

If the `Queue` here is destroyed half-way while a workflow is running because of a crash, what will happen to the prints in it? Will they be lost forever, or will we deterministically recreate them in the new `Queue` based on the workflow history from Temporal?